src/components/internals/search-icon-dropdown-indicator/search-icon-dropdown-indicator.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { SearchIcon } from '@commercetools-uikit/icons';
+import { SearchIconWrapper } from './search-icon-dropdown-indicator.styles';
+
+const handleMouseDown = (event, props) => {
+  event.stopPropagation();
+  event.preventDefault();
+  return props.selectProps.onSearchIconClick(props.selectProps.inputValue);
+};
+
+const SearchDropdownIndicator = (props) => {
+  return (
+    <SearchIconWrapper
```

Comment:
We should make sure this is accessible.

You can use the `AccessibleButton` component as the wrapper and use `onClick` instead of onMouseDown 🙂

Reply:
Used that but the the custom handle for the `onClick` of `AccessibleButton` does not stop the event propagation, even though if you use `event.stopPropagation(); event.preventDefault();`

Reply:
Ah, I think react-select uses the onMouseDown event handler internally.
Can we stopPropagation & preventDefault for that event and still use the onClick?